src/Uno.UI/ts/Windows/System/Profile/AnalyticsInfo.ts

```diff
@@ -0,0 +1,62 @@
+﻿namespace Windows.System.Profile {
+
+	export class AnalyticsInfo {
+		public static getDeviceType(): string {
+
+			// Logic based on https://github.com/barisaydinoglu/Detectizr
+
+			var ua = navigator.userAgent;
+
+			if (!ua || ua === '') {
+				// No user agent.
+				return "unknown";
+			}
+			if (ua.match(/GoogleTV|SmartTV|SMART-TV|Internet TV|NetCast|NETTV|AppleTV|boxee|Kylo|Roku|DLNADOC|hbbtv|CrKey|CE\-HTML/i)) {
```

Comment:
Codacy found an issue: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6396919)